dot/state/block_test.go

```diff
@@ -543,3 +547,53 @@ func TestNumberIsFinalised(t *testing.T) {
 	require.NoError(t, err)
 	require.False(t, fin)
 }
+
+func TestBlockState_GetBestBlockRuntime(t *testing.T) {
+	blockState := newTestBlockState(t, newTriesEmpty())
+	cfg := wasmer.Config{}
+	polkadotRuntimeFilepath, err := runtime.GetRuntime(
+		context.Background(), runtime.POLKADOT_RUNTIME)
+	require.NoError(t, err)
+	code, err := os.ReadFile(polkadotRuntimeFilepath)
+	require.NoError(t, err)
```

Comment:
nit can you use a mock instance to store in the runtimes instead of a full instance? To reduce the resource usage / time it takes to init.

Reply:
I didn't want to use a mock for blockState since then the test would then be just testing the mock and not the way that runtime is actually stored.